.kci-dev.toml.example

```diff
@@ -2,20 +2,15 @@ default_instance="local"
 
 [local]
 host="https://127.0.0.1"
+api="https://127.0.0.1:8001/"
 token="example"
 
-[staging_pipeline]
+[staging]
 host="https://staging.kernelci.org:9100/"
```

Comment:
what about 
```
[staging]
pipeline_host=""
api_host=""

[production]
pipeline_host=""
api_host=""
```
or
```
[staging]
pipeline=""
api=""

[production]
pipeline=""
api=""
```

Reply:
Mixing host and api makes it confusing. as api could refer to api version or api framework

Reply:
Agree, will update PR